lib/tasks/dev.rake

```diff
@@ -9,10 +9,9 @@ if Rails.env.development? || Rails.env.test?
       user = create(:user, email: 'user@example.com')
 
       dashboard_saml_config = Saml::Config::SETTINGS
-      uuid = SecureRandom.uuid
```

Comment:
I just ran this locally and got an error because `uuid` (used in the `metadata_url`) is now undefined:

```
NameError: undefined local variable or method `uuid' for main:Object
```